mempool/reactor.go

```diff
@@ -67,11 +78,18 @@ func (memR *MempoolReactor) GetChannels() []*p2p.ChannelDescriptor {
 // AddPeer implements Reactor.
 // It starts a broadcast routine ensuring all txs are forwarded to the given peer.
 func (memR *MempoolReactor) AddPeer(peer p2p.Peer) {
+	memR.idMtx.Lock()
+	memR.peerMap[peer.ID()] = memR.nextID
+	memR.nextID++
```

Comment:
what about peers who constantly drop / reconnect? what will happen when we go beyond 65k?

Reply:
Thats moreso an argument to just not remove them from the map, which seems like a good idea to me!

Reply:
> But what if we're spammed by 65k peers? Won't we overflow?

That was my question basically.

Reply:
A reverse map sounds good to me